### ISSUE TITLE
Unused lints don't trigger for tuples.

### ISSUE THREAD

**Issue by vloddot** [charliermarsh/ruff#4577]
### The bug
Tuples don't produce unused lints.

For example, this lints as unused:

```python
a = input()
```

However, this doesn't:

```python
a, b = input().split(' ')
```

### Command:
```sh
ruff <filename> --isolated
```

### Ruff settings
Default.

### Ruff version
ruff 0.0.269

**Comment by charliermarsh** [comment]
This is consistent with Flake8 and Pyflakes behavior, and I believe is intentional:

- https://github.com/PyCQA/pyflakes/issues/143
- https://github.com/PyCQA/pyflakes/issues/341

**Comment by charliermarsh** [comment]
Note that Ruff and Pyflakes _will_ flag if the RHS is a tuple:

```python
def f():
    c, d = (2, 3)

```

**Comment by charliermarsh** [comment]
We could perhaps try to flag `a` and `b` if _all_ tuple members are unused, but it adds enough complexity that I'm going to mark this as "expected" for now.

**Comment by rodya-mirov** [comment]
Is there another lint we can add that _will_ trigger on this? I see this is consistent with pyflakes, but it's undesirable (to me) and I know there are quite a few other lint sets available. Google is only showing this one issue, so not sure where else to ask.
